apps/frontend/src/app/OrderDetails/OrderDetails.tsx

```diff
@@ -1,7 +1,209 @@
-import { FC } from 'react';
+import type { OrderModel } from '@pasnik/api/data-transfer';
+import { OrderStatus } from '@pasnik/api/data-transfer';
+
+import { FC, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Chip,
+  Container,
+  createTheme,
+  CssBaseline,
+  Grid,
+  TextField,
+  ThemeProvider,
+  Typography
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import { DishModel } from '../../../../../libs/api/data-transfer/src/lib/dish.model';
+
+const initDate = new Date('2015-03-25T12:00:00Z');
+
+//TODO: replace mockOrder with real order
+const mockOrder: OrderModel = {
+  createdAt: initDate,
+  from: 'Order Name',
+  dishes: [
+    { name: 'mockDish 1', priceCents: 2150 },
+    { name: 'mockDish 2', priceCents: 1030 },
+    { name: 'mockDish 3', priceCents: 3820 }],
+  id: '',
+  shippingCents: 300,
+  menuUrl: 'example.com',
+  orderedAt: '',
+  status: OrderStatus.InProgress,
+  updatedAt: initDate,
+  user: { email: 'example@example.com' }
+};
+
+function Dish(props: { dish: { name: string, priceCents: number } }) {
+  return (
+    <Grid
+      container
+      style={{
+        paddingTop: '1rem'
+      }}
+      spacing={2}
+      alignItems='baseline'
+      justifyContent='space-between'
+    >
+      <Grid item xs={8} sm={9} style={{ padding: '0' }}>
+        <div style={{
+          border: '1px #9ca3af solid',
+          backgroundColor: 'white',
+          borderRadius: '0.125rem',
+          fontWeight: 'bolder',
+          marginTop: '0.5rem'
+        }}>
+          <div style={{ marginLeft: '0.2rem', padding: '0.2rem' }}>
+            {props.dish.name}
+          </div>
+        </div>
+      </Grid>
+      <Grid item xs={4} sm={3} style={{ paddingTop: '0' }}>
+        <div style={{
+          border: '1px #9ca3af solid',
+          backgroundColor: 'white',
+          borderRadius: '0.125rem',
+          fontWeight: 'bolder',
+          marginTop: '0.5rem'
+        }}>
+          <div style={{ marginLeft: '0.2rem', padding: '0.2rem' }}>
+            {PriceFormatter(props.dish.priceCents)}
+          </div>
+        </div>
+      </Grid>
+      <Grid item xs={2} sm={2} style={{ paddingTop: '0' }}>
+      </Grid>
+    </Grid>
+  );
+}
+
+function DishList(props: { dishes: { name: string; priceCents: number; }[]; }) {
```

Comment:
this is a component or helper function? if it's a helper fn please use camel case naming + please add a proper type